libs/wingii/src/util.rs

```diff
@@ -86,3 +74,56 @@ pub mod package_json {
 		}
 	}
 }
+
+#[cfg(test)]
+mod tests {
+	use super::*;
+	use serde_json::json;
+	#[test]
+	fn test_dependencies_of() {
```

Comment:
do we want to test some malformed json cases?

Reply:
You mean like invalid json text? Sure, but everything in this file already assumed a successfully parsed file (`serde_json::Value`) so going any further out isn't in scope for this pr fix.